apps/src/lab2/responseValidators.ts

```diff
@@ -46,6 +46,21 @@ const PythonSourceResponseValidator: ResponseValidator<
   return sourceValidatorHelper(response, pythonValidator);
 };
 
+// Validator for labs that use JSON sources
+const JsonSourceResponseValidator: ResponseValidator<
```

Comment:
nit: technically, Blocky sources are also json sources I think.

Reply:
Good call, will update the comment to specify